ansible_collections/arista/avd/roles/eos_designs/templates/connected_endpoints/port-channel-interfaces.j2

```diff
@@ -88,6 +95,27 @@ port_channel_interfaces:
 {%                         if adapter_settings.port_channel.structured_config is arista.avd.defined %}
     struct_cfg: {{ adapter_settings.port_channel.structured_config }}
 {%                         endif %}
+{# Render Port-Channel subinterfaces #}
+{%                         for subinterface in adapter_settings.port_channel.subinterfaces | arista.avd.natural_sort %}
+{%                             if subinterface.number is arista.avd.defined %}
+  Port-Channel{{ channel_group_id }}.{{ subinterface.number }}:
+    type: l2dot1q
+    mode: flexencap
```

Comment:
Is this supported in eos_cli_config_gen

Reply:
Correct, this should not be there. We will fix.